modules/ocl/cv_calibration_parser.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * cv_calibration_parser.h - parse fisheye calibration file
+ *
+ *  Copyright (c) 2016-2017 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Author: Junkai Wu <junkai.wu@intel.com>
+ */
+
+#ifndef XCAM_CV_CALIBRATION_PARSER_H
+#define XCAM_CV_CALIBRATION_PARSER_H
+
+#include <base/xcam_common.h>
+#include <base/xcam_buffer.h>
+#include <dma_video_buffer.h>

```

Comment:
1. all xcam c++ header files start from "xcam_utils.h"
2. I didn't see xcam_buffer.h and dma_video_buffer.h is necessary to use. 
Header files only include necessary headers.